velox/exec/TimSort.hpp

```diff
@@ -0,0 +1,814 @@
+/*
```

Comment:
Can we have any basic testing on this utility? Also a benchmark to show the performance difference? Thanks!